backend/integrations/discord/admin_cog.py

```
@@ -0,0 +1,411 @@
+            for name, count in cleared.items():
+                embed.add_field(name=name.replace("_", " ").title(), value=str(count), inline=True)
+
+            if cache_type == "all":
+                await interaction.edit_original_response(embed=embed, view=None)
+            else:
+                await interaction.followup.send(embed=embed, ephemeral=True)
+
+        except Exception as e:
+            logger.error(f"Cache clear failed: {e}", exc_info=True)
+            if interaction.response.is_done():
+                await interaction.followup.send(f"Clear failed: {str(e)}", ephemeral=True)
```

Ruff (0.15.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)